bot/exts/utils/snekbox/_cog.py

```diff
@@ -373,11 +382,39 @@ async def send_job(self, ctx: Context, job: EvalJob) -> Message:
 
                 msg += f"\n{Emojis.failed_file} {blocked_msg}"
 
+            # Split text files
+            text_files = [f for f in allowed if f.suffix in TXT_LIKE_FILES]
+            # Inline until budget, then upload to paste service
+            budget_lines = MAX_OUTPUT_BLOCK_LINES
+            budget_chars = MAX_OUTPUT_BLOCK_CHARS
```

Comment:
This budget is shared with the stdout.

Reply:
Updated to be shared in [4fed441](https://github.com/python-discord/bot/pull/2326/commits/4fed4410ea9f0087f5c60c66c1291adffa7cd896)